lib/idx/remediators/Base/Remediator.ts

```diff
@@ -182,6 +182,10 @@ export class Remediator {
       if (Array.isArray(input)) {
         input.forEach(i => inputs.push(i));
       } else {
+        // guarantees field-level messages are passed back
+        if (messages) {
+          input.messages = messages;
```

Comment:
field-level messages fix

Reply:
I remember we also have a jira for this issue, we can close that one together.